src/utils/useAxios/UseAxios.jsx

```diff
@@ -0,0 +1,25 @@
+import { useState } from "react";
+import axios from "axios";
+
+axios.defaults.baseURL = "";
+
+export const useAxios = () => {
+  const [response, setResponse] = useState(undefined);
+  const [error, setError] = useState("");
+  const [loading, setLoading] = useState(false);
+
+  const operation = async (params) => {
+    try {
+      setLoading(true);
+      const result = await axios.request(params);
+      setResponse(result.data);
+    } catch (error) {
+      console.log("axios: ", error);
```

Comment:
You could remove console log here

Reply:
Yes, I will do it. Thanks for pointing it out.